jobs/service-fabrik-broker/templates/config/circuit-breaker-config.yml.erb

```diff
@@ -0,0 +1,24 @@
+<% if_p('broker.circuit_breaker') do |config| %>
+<%= JSON.dump(config) %>
+<% end.else do %>
+# Circuit breaker configurations are done at Protocol level (only http for now)
+# If an API URL is not configured in the below, then all calls to that subsystem bypass the circuit breaker.
+# One circuit is created for each of the configured API URL (Base Circuit) & additional circuits are created if configuration are overridden.
+# Appropriate circuit is chosen from bottom-up i.e. if for a particular path in API if configuration is overridden, then corresponding circuit with overriden config is applied, else Base circuit is applied)
+http:
+  error_threshold_percentage: 49 #error percentange at which the circuit trips
```

Comment:
circuit breaker configuration is either defined entirely at landscape & if not defined, these are the defaults. So if defined at landscape level its a json dump of entire prop set. The configuration is such that overriding can happen at any level, so defining properties for each of them in spec will be really huge and hence adopted the above approach of JSON dump from manifest